flashinfer/topk_varlen/kernels/config.py

```
@@ -0,0 +1,195 @@
+    hardware-aware auto-selection of every knob).
+
+    Parameters
+    ----------
+    cluster_size : int
+        Number of CTAs per thread-block cluster.  ``1`` → single-CTA path;
+        ``2`` or ``4`` enable DSMEM aggregation for very long rows (> 64 K).
+        B200 GPC limit caps usable values at ~16.
+    num_threads_per_block : int
+        CTA block size.  Valid values: ``512`` or ``1024``.
+    enable_unroll_4 : bool
+        Unroll Phase-2/3 inner scan loop 4× for LSU-pipelining ILP.
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)